README.md

```diff
@@ -11,16 +11,19 @@ Rust KeePass database file parser for KDB, KDBX3 and KDBX4, with experimental su
 
 ## Example
 ```rust
-use keepass::{Database, DatabaseOpenError, NodeRef};
+use keepass::{
+    db::{Database, NodeRef},
+    error::DatabaseOpenError,
+    Key
```

Comment:
@sseemayer how about `DatabaseKey`? That would be consistent with the other public struct we expose.

Reply:
Agreed and adjusted accordingly.